Install.bat

```diff
@@ -1,3 +1,7 @@
+mkdir secrets
+cd secrets
+"GUILD_ID=\nTOKEN=\nCLIENT_ID=" > .env
```

Comment:
würd die .env file so lassen
später wird das zeugs eh in der Dockerfile gesetzt